recipes/_user.rb

```diff
@@ -44,10 +44,13 @@
   only_if { solaris_10? }
 end
 
-group node['omnibus']['build_user_group'] do
+group "create #{node['omnibus']['build_user_group']} group" do
+  group_name node['omnibus']['build_user_group']
   # The Window's group provider get's cranky if attempting to create a
   # built-in group.
   ignore_failure true if windows?
+  # Don't remove existing members - in case this is a built-in group.
+  append true 
```

Comment:
I think it's slightly weirder.  append true when used with modify doesn't delete existing members from a group but doesn't create the group if it doesn't exist.  append true when used with create will not remove existing members from the group if it doesn't exist but the windows provider is whacky and errors if you try to create a preexisting group (like Administrators).  So we attempt to create the group while ignoring errors and then to add to the group.

The weird error you were seeing before is due to the fact that we had two resources that were named the same thing - and so their properties got cloned from one to the other and hilarious bad things happened but the error was being ignored.